docs/example/manifests/apps/bookstore.yaml

```diff
@@ -16,8 +16,6 @@ spec:
   ports:
   - port: 80
     name: bookstore-port
-  selector:
-    app: bookstore
```

Comment:
We need to retain this to demo permissive mode with the current manual demo documentation.